common/constants.go

```diff
@@ -0,0 +1,20 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+const (
+	// TestEnvNameURI is a key for connection uri.
+	TestEnvNameURI = "CONNECTION_URI"
```

Comment:
Given that it's only one constant and that it's used in tests, I think we should declare it in a test file.

Reply:
Even if it uses in different files?
`acceptance_test.go`, `/source/source_integration_test.go`, `/destination/destination_integration_test.go`

Reply:
I think so because it doesn't "feel" right that a test constant lives in the "production" code. We could put it into a common test file, but on the other hand that's just one constant, used 3 times. Theoretically, the integration tests and acceptance don't even have to use the same instance. That said, I don't think it's a blocker to this PR. I'm completely fine that the PR is merged as it is.